.circleci/config.yml

```diff
@@ -144,30 +144,36 @@ commands:
       study_key:
         type: string
     steps:
+      - run:
+          name: Setup gcloud context
+          command: |
+            set -u
+            readvault.sh secret/pepper/${ENVIRONMENT}/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
+            gcloud --quiet config set compute/zone ${GOOGLE_COMPUTE_ZONE}
       - run:
```

Comment:
Moved logging into GCP ahead of generating config files as DSM uses GCP secrets to create its config file.